MassEditConnector/JobParameters/DefaultValuesProvider/QuickCsvExport.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Pim\Bundle\CustomEntityBundle\MassEditConnector\JobParameters\DefaultValuesProvider;
+
+use Akeneo\Component\Batch\Job\JobInterface;
+use Akeneo\Component\Batch\Job\JobParameters\DefaultValuesProviderInterface;
+
+/**
+ * @author    Nicolas Dupont <nicolas@akeneo.com>
+ * @copyright 2016 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class QuickCsvExport implements DefaultValuesProviderInterface
```

Comment:
Same than for EnhancedConnectorBundle, I am not convinced about this naming. QuickCsvExport class names will be extract many times for autocompletion.